cli/cmd/logs.go

```diff
@@ -10,13 +11,32 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var logsDescription = "Check Service logs. Requires one Service: " + servicesList()
+
 var LogsCmd = &cobra.Command{
-	Use:     "logs",
-	Short:   "Check service logs",
+	Args: func(cmd *cobra.Command, args []string) error {
+		_, found := controller.Services[args[0]]
```

Comment:
This wil panic, because you are accessing the args[0] which is undefined. better check len(args) != 1 to return error.